packages/rbac/v2/tests/integration/scripts/serve.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+echo "Running API using the updated spec"
+npx mockoon-cli start --data /tmp/workspaces_updated.json
```

Comment:
@catastrophe-brandon I think we can follow up to make this more reusable/put this script in a shared location outside of rbac.